apis/.schemes/core-v1alpha1-Blueprint.json

```diff
@@ -0,0 +1,434 @@
+          "description": "SecretRef defines a data reference from a secret. This method is not allowed in installation templates.",
+          "$ref": "#/definitions/core-v1alpha1-SecretReference"
+        },
+        "version": {
+          "description": "Version specifies the imported data version. defaults to \"v1\"",
+          "type": "string"
+        }
+      }
+    },
+    "core-v1alpha1-Default": {
+      "description": "Default defines a default value (future idea: also reference?).",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "string",
+          "format": "byte"
+        }
+      }
+    },
+    "core-v1alpha1-ExportDefinition": {
+      "description": "ExportDefinition defines a exported value",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "name": {
+          "description": "Name defines the field name to search for the value and map to exports. Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#selecting-fields",
+          "type": "string",
+          "default": ""
+        },
+        "schema": {
+          "description": "Schema defines the imported value as jsonschema.",
+          "type": "string",
+          "format": "byte"
+        },
+        "targetType": {
+          "description": "TargetType defines the type of the imported target.",
+          "type": "string"
+        }
+      }
+    },
+    "core-v1alpha1-ImportDefinition": {
+      "description": "ImportDefinition defines a imported value",
+      "type": "object",
+      "required": [
+        "name"
+      ],
+      "properties": {
+        "default": {
+          "description": "Default sets a default value for the current import that is used if the key is not set.",
+          "default": {},
+          "$ref": "#/definitions/core-v1alpha1-Default"
+        },
+        "imports": {
+          "description": "ConditionalImports are Imports that are only valid if this imports is satisfied. Does only make sense for optional imports.",
+          "type": "array",
+          "items": {
+            "default": {},
+            "$ref": "#/definitions/core-v1alpha1-ImportDefinition"
+          }
+        },
+        "name": {
+          "description": "Name defines the field name to search for the value and map to exports. Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#selecting-fields",
+          "type": "string",
+          "default": ""
+        },
+        "required": {
+          "description": "Required specifies whether the import is required for the component to run. Defaults to true.",
+          "type": "boolean"
+        },
+        "schema": {
+          "description": "Schema defines the imported value as jsonschema.",
+          "type": "string",
+          "format": "byte"
```

Comment:
incorrect schema definition. an import in the form
```
- name: ingressClass
  schema:
    type: string
```

should be allowed. (same for the export)

Maybe a problem resolving the  json.RawMessage